guides/common/header.adoc

```diff
@@ -6,10 +6,22 @@
 :toclevels: 3
 ifndef::satellite[]
 :docinfodir: common
+++++
+<!-- "Fork me on GitHub" CSS ribbon v0.2.3 | MIT License -->
+<!-- https://github.com/simonwhitaker/github-fork-ribbon-css -->
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/github-fork-ribbon-css/0.2.3/gh-fork-ribbon.min.css" />
```

Comment:
I would prefer to have this locally commited into foreman-documentation. Similar to https://github.com/theforeman/foreman-documentation/pull/3453#pullrequestreview-2435248034 and https://github.com/theforeman/foreman-documentation/pull/3453#discussion_r1842722163

Reply:
Ok, I'll look into it. As long as we credit the source properly, it should be alright.

Reply:
Resolved